README.md

```diff
@@ -1,3 +1,29 @@
 # fabric8-gemini-server
 Fabric8-server analytics powered services to initiate and report the readiness of
 all registered services
+
+### ENDPOINTS
+
+#### POST: /api/v1/register
+```
+{
+	"git_url" : <some_url>,
+	"git_sha" : <some_sha>,
+	"email_ids" : <some_email>
+}
+
+```
+
+##### Result :
+```
+{
+  "data": {
+    "email_ids": <some_email>,
+    "git_sha": <some_sha>,
+    "git_url": <some_url>,
+    "last_scanned_at": "Mon, 05 Mar 2018 00:00:00 GMT"
+  },
+  "success": true,
```

Comment:
I am going to use `success` field to decide if `POST` request from client went well.

@GeetikaBatra : Anything around HTTP status code the POST API could return with?

Reply:
HTTP for success is `200 OK`. Will update the Read me for the same.